docs/style_guide/README.md

```diff
@@ -0,0 +1,78 @@
+# Ballerina Style Guide
+
+Formatting is one of the most argued topic when it comes to a programming language.
+To avoid these arguments we decide to come up with a opinionated guide for Ballerina source code formatting. 
+Users can follow their own formatting but we recommend to use this formatting guide as all the IDE tools 
+will support only this format. Also following this guide will form a standard style across all developers.
+
+## Formatting Conventions
+
+### Indentation and line length
+* Use four spaces, not tabs
+* No indentation top level (definitions) 
+* Maximum line length is 120 characters 
+
+### [Operators, Keywords and Boundaries](operators_keywords_and_boundaries.md)
+### [Definitions](definitions.md)
+### [Statements](statements.md)
+### [Expressions](expressions.md)
+
+### Comments (//)
+
+* If Comment is taking an entire line it should indent according to where it is.
```

Comment:
Shall we use some other term instead of _**according to where it is.**_

Reply:
will fix it.